data/json/professions.json

```diff
@@ -3379,5 +3379,37 @@
                 "hardtack"
             ]
         }
+    },{
+
+        "type": "profession",
+        "ident": "drone_op",
+        "name": "Drone Operator",
+        "description": "You had a job programming machines like automatic street cleaners, newsbots and pizza delivery drones. Now the all the drones carry guns instead of pizza.",
```

Comment:
Beat PD actually has been largely phased out for drones, IIRC.  Pizza and paper delivery that requires human intervention would probably be more along the lines of a marketing thing: Know Your Deliverator.